AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLSyncBased/GraphQLSyncBasedTests-amplifyconfiguration.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This should not have been committed with actual content. 

1. obfuscate the content of this file on the next commit
1. `amplify delete` the project that generated these resources and re-generate a new one

Reply:
completed this by updating top level .gitignore with `*amplifyconfiguration.json` from `amplifyconfiguration.json` and removed individual .gitignores that i created. deleted and recreated the resources/reuploaded new files